benchto-driver/src/test/java/io/trino/benchto/driver/DriverAppIntegrationTest.java

```diff
@@ -89,11 +101,11 @@ public void testBenchmark()
     @Test
     public void testConcurrentBenchmark()
     {
-        ImmutableList<String> concurrentQueryMeasurementName = ImmutableList.of("duration");
-        ImmutableList<String> concurrentBenchmarkMeasurementNames = ImmutableList.<String>builder()
+        List<Measurement> concurrentQueryMeasurement = ImmutableList.of(measurement("duration", "MILLISECONDS", 0.0, ImmutableMap.of("scope", "driver")));
```

Comment:
It looks like a factory method. Maybe something like `Measurement.of`?

Reply:
I agree, but I'd rather do this in a separate PR.